src/Compilers/CSharp/Portable/Parser/Lexer_RawStringLiteral.cs

```diff
@@ -0,0 +1,347 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Text;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.CSharp.Syntax.InternalSyntax
```

Comment:
nit: depending how fresh the branch is, you could use file-scope namespaces for new files ;-)

Reply:
i really would prefer to not mix styles in teh compiler layer.  It's really hard editing here because everything is very different per file (or even per method).  If we could add some tooling ot make things consistent i would be much more amenable to then moving over to these styles :)